operations/experimental/enable-routing-integrity.yml

```diff
@@ -9,4 +9,4 @@
 
 - type: replace
   path: /instance_groups/name=router/jobs/name=gorouter/properties/router?/ca_certs
-  value: ((application_ca.ca))
+  value: ((application_ca.certificate))
```

Comment:
Yeah, this value would make sense. The other logical choice would be to use `((diego_instance_identity_ca.ca))`, but https://github.com/cloudfoundry/bosh-cli/issues/354 (and the underlying https://github.com/cloudfoundry/config-server/issues/5) end up producing the wrong CA cert with the BOSH CLI.